xc/emu_manager.ml

```diff
@@ -201,13 +201,15 @@ let receive_success ?(debug_callback=(fun s -> debug "%s" s)) cnx =
 
 let with_connection (task: Xenops_task.task_handle) path domid (args: string list) (fds: (string * Unix.file_descr) list) f =
   let t = connect path domid args fds in
+  let send_abort cnx = send cnx "abort\n" in
```

Comment:
Perhaps put this on top with the other `send_` functions? It may become useful outside of here.